### PR TITLE
New method - get_media_group() 

### PR DESCRIPTION
**Pull request by Legenda24**
Method to get all media group messages knowing only one. Special thank to Danipulok (https://github.com/Danipulok) 

"Problems":
1) Docs
Doubt how to write correctly so I marked those lines with  "#TODO" + examples. 

2) Should it raises error in case it's passed "wrong" (not related to media group) message_id or just return an empty list? 

UPD(5.12.2020):
I just noticed that other media groups have media_group_id attribute too (playlists etc.)
So now I'm 100% sure this method supports all types of media groups(playlists, albums etc.)  as it should be :)

### CODE REVIEW

**Comment by Danipulok** [comment]
Well, I can say that we were testing a few options and it was the fastest one and pretty the easiest one to read.

And I also think that the name should be _get_album()_, because if there's method that will get all the files in one file group (like a few audiofiles together or a file .txt or other files) that one should be called _get_media_group()_.

File: pyrogram/methods/messages/get_media_group.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            #TODO ValueError: In case message isn't related to media group. 
+        """
+        messages = await self.get_messages(chat_id, [msg_id for msg_id in range(message_id-9, message_id+10)], replies=0)
+
+        if messages[9].media_group_id:
+            media_group_id = messages[9].media_group_id
+        else:
+            raise ValueError("Message isn't related to media group") #TODO
+
+        return list.List(msg for msg in messages if msg.media_group_id == media_group_id)
```
Comment (delivrance) [inline pyrogram/methods/messages/get_media_group.py:56]: You can import `List` directly from `pyrogram.types`

File: pyrogram/methods/messages/get_media_group.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            List of :obj:`~pyrogram.types.Message`
+        Raises:
+            #TODO ValueError: In case message isn't related to media group. 
+        """
+        messages = await self.get_messages(chat_id, [msg_id for msg_id in range(message_id-9, message_id+10)], replies=0)
+
+        if messages[9].media_group_id:
+            media_group_id = messages[9].media_group_id
+        else:
+            raise ValueError("Message isn't related to media group") #TODO
```
Comment (delivrance) [inline pyrogram/methods/messages/get_media_group.py:54]:
``` python
media_group_id = messages[9].media_group_id

if media_group_id is None:
    raise ...
```

looks better.

File: pyrogram/methods/messages/get_media_group.py (hunk 1 above)
Comment (Legenda24) [reply pyrogram/methods/messages/get_media_group.py:56]: Yes, but it will be mismatching between typing List and pyrogram List

Comment (delivrance) [reply pyrogram/methods/messages/get_media_group.py:56]: Not really, pyrogram's List is an actual list, it exists only for pretty print purposes. The only issue I see here is that you have to rename either one so it's ok to keep it like so.

Comment (Legenda24) [reply pyrogram/methods/messages/get_media_group.py:56]: I am better off with the present state of things.

**Review by delivrance — CHANGES_REQUESTED** [review]
All is good. Could you add the friendly bound method `Message.get_media_group` as well?
chat and message id will be taken from the message instance itself, so this method will have no parameter.

**Comment by Legenda24** [comment]
Don't sure, so more like no. Tbh this is my very first experience "contributing" smth in open-source project ;)
I'd better leave this job for more "knowing" people

**Pull request merged**